CHANGELOG.md

```diff
@@ -773,6 +774,7 @@ By @MarijnS95 in [#6006](https://github.com/gfx-rs/wgpu/pull/6006).
 
 #### General
 
+- Added an example that shows how to handle datasets too large to fit in a single `GPUBuffer` by distributing it across many buffers, and then having the shader receive them as a `binding_array` of storage buffers.  By @alphastrata in [#6138](https://github.com/gfx-rs/wgpu/pull/6138)
```

Comment:
Before we merge, can this be moved to the new `Unreleased` section?

Reply:
Yep, ETA 2, 3 hrs.